docs/deployments/aws/ecs-update-service/index.md

```diff
@@ -0,0 +1,140 @@
+
+![Update ECS Step Container Definitions Section](images/update-ecs-container-definitions.png "width=500")
+
+Specify the container name that will be used to locate the container definition within your task and select a feed and image that will be run by your task.
+The specific image version will be specified later, when creating a release.
+
+![ECS Step Container Definition Parameters](images/update-ecs-container-definition.png "width=500")
+
+#### Environment Files
+
+List up to 10 environment files to be added to the container.
+Environment files should be stored in S3, they must have a ".env" extension, and each line should contain a "VARIABLE=VALUE" variable definition.
+
+The configured environment file list can be used to either completely replace or be appended to the container's existing environment file list.
+
+Each environment file has a type to configure the storage device (S3 is currently the only option) and a source to specify the location within that storage (for S3, this is an ARN).
+
+![Update ECS Step Container Definition Environment Files](images/update-ecs-environment-files.png "width=500")
+
+#### Environment Variables
+
+You can add either plain text or secret environment variables to the container.
+Plain text environment variables will use the value provided, while secret variables refer to a secret value stored in the AWS Secrets Manager.
+
+The configured environment variables can either completely replace the container's existing environment variables or be merged with them.
+Merging environment variables will add any new variables and overwrite the value of any existing variables with the same key.
+
+![Update ECS Step Container Definition Environment Variables](images/update-ecs-environment-variables.png "width=500")
+
+### Tags section
+
+Specify any additional tags that should be added to the task definition and service. Tags can be used to attach arbitrary metadata to categorize and organize resources. For more information see the [ECS Documentation](https://g.octopushq.com/ECSUsingTags)
+
+![ECS Step Tags](images/update-ecs-tags.png "width=500")
+
+### Deployment Options section
+
+You can optionally change whether the step should wait until the release is fully deployed by changing the `Wait Option` selection.
+By default, the step will wait until all of the ECS Tasks within the target service are running (or failed to run).
+
+![ECS Step Deployment Options section](images/update-ecs-deployment-option.png "width=500")
+
+:::hint
+Selecting the `Don't wait` option means that the step will not fail if the tasks fail to properly start.
+:::
+
+#### Variable replacements
+
+Any of the input fields can be bound to an Octostache expression. [Variable substitution](/docs/projects/variables/variable-substitutions.md) will be performed before the release is deployed.
+
+### Output variables
+
+Presently, the step does outputs the following variables.
+
+TaskDefinitionRevision
```

Comment:
I think these variables would be better placed in a table. Something like this:

```
| Name | Description |
| -------------------- | -------|
|`TaskDefinitionRevision` | The revision number of the newly created task (if a new revision was created) |
|`TaskDefinitionFamily` | The name of a family that the new task definition is registered to (if a new revision was created) |
|`ClusterName` | The name of the cluster to which the updated service belongs |
|`Region` | The AWS region in which the operations were executed |
```
That would output like this:

| Name | Description |
| -------------------- | -------|
|`TaskDefinitionRevision` | The revision number of the newly created task (if a new revision was created) |
|`TaskDefinitionFamily` | The name of a family that the new task definition is registered to (if a new revision was created) |
|`ClusterName` | The name of the cluster to which the updated service belongs |
|`Region` | The AWS region in which the operations were executed |

Reply:
Yep, I did consider making it a table but I didn't know how people felt about them. The format I used is supposed to be a definition list but it's an extended markdown syntax and I wasn't sure if our parser/renderer supports it.

I'll change it for a table.